projboard/tests/test_user.py

```diff
@@ -49,3 +55,8 @@ def test_delete_user_by_nickname(self, generate_user):
         # Delete user by nickname
         generate_user.delete_user_by_nickname(generate_user.nickname)
         assert generate_user not in User.objects.all()
+
+    @pytest.mark.django_db
+    def test_delete_user_by_nickname_fail(self, fake_user):
```

Comment:
same questions as previous one

Reply:
What does the tests do?
test_delete_non_exist_user?
In that case no need to use django-pytest fixtures, you can use pytest fixture that generates only the name.